src/components/Feeds/FeedPage/FeedComments/AddFeedComment.jsx

```diff
@@ -24,9 +24,9 @@ const AddFeedComment = ({ feedId, userDetails }) => {
             return;
         }
 
-        const replyData = { reply };
-        dispatch(addFeedComment({ feedId, replyData }));
-        setAddReply({ reply: "" });
+        const replyData = { content };
+        dispatch(createFeed({ feedData: replyData, parentId: feedId }));
```

Comment:
Correct me if I'm wrong but you are using `content` just to assign it into `replyData`? 

Could you for example do something like this `const { replyData }= addReply `

Do you think it will it work or I`m missing something ?

Reply:
![image](https://github.com/thecyberworld/TheCyberHUB/assets/76536506/ad0852a6-f646-40d1-87a1-fb31230555e3)

We use it to pass to FeedCommentInput as well